app/jobs/access_copy_job.rb

```diff
@@ -0,0 +1,24 @@
+# A job to apply work permissions to all contained files set
+#
+class AccessCopyJob < ActiveJob::Base
```

Comment:
I'm not sure about the name. This is copying permissions down into child resources, right? What about PropagatePermissionsJob or InheritPermissions? InheritAccess?